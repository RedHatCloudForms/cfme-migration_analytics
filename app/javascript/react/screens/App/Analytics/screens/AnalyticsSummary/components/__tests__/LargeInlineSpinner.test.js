// TODO add unit tests for LargeInlineSpinner.js

test.todo('add unit tests here');
