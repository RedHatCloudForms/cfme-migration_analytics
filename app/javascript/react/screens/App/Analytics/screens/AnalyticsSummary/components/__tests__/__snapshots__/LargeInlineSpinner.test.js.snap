// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`large inline spinner renders correctly 1`] = `
<div
  className="large-spinner"
>
  <Spinner
    className=""
    inline={true}
    inverse={false}
    loading={true}
    size="lg"
  />
  <h3>
    Loading stuff
  </h3>
</div>
`;
